android/gltfio-android/settings.gradle

```diff
@@ -0,0 +1,12 @@
+/*
+ * This file was generated by the Gradle 'init' task.
+ *
+ * The settings file is used to specify which projects to include in your build.
+ * 
+ * Detailed information about configuring a multi-project build in Gradle can be found
+ * in the user guide at https://docs.gradle.org/4.6/userguide/multi_project_builds.html
+ */
+
+includeBuild '../filament-android'
```

Comment:
This means that all filament-android will be built into this library. That makes sense for sample apps, probably not for a library?

Reply:
Yes good catch Romain. I need to figure out how to link Filament dynamically and avoid "unresolved externals".  I think I need to point the CMake file to `libfilament-jni.so` somehow (we don't have existing examples of this because filamat does not depend on filament)

Let me look into it.  (I made an attempt earlier today and failed.)

Reply:
In response to this comment, I'll change this project's CMake to use STATIC instead of SHARED for all upstream library.

However I'm unable to remove this particular line from the settings gradle file, because then the Java compiler fails to find the Filament classes...

Reply:
Then that's a problem: you are compiling the entire Filament library into gltfio. This means the user cannot add gltfio to a project that already links against Filament, there will be duplicate classes and that will lead to a class loading error. You need to make gltfio link against the main FIlament AAR.

Reply:
Apparently not, after looking inside the AAR. I'm not sure I understand why, but it's a good thing. The gltfio .so libs are big though (1.2 MiB for armv7a), is it just because of materials/cgltf or do they include Filament itself?